py/picca/tests/delta_extraction/expected_flux_tests.py

```diff
@@ -1667,7 +1667,7 @@ def test_true_continuum_populate_los_ids(self):
 
         # compute the forest continua
         for forest in data.forests:
-            expected_flux.read_true_continuum(forest)
+            expected_flux.read_true_continuum([forest])
```

Comment:
can we add a test for reading a group of forests with the same healpix? The way we now do it on the code?

Reply:
Not sure if I did it what you wanted here. I put all "reading a group of forests" lines into another function. Now I call that function in tests.